### ISSUE TITLE
Support for API Gateway requests?

### ISSUE THREAD

**Issue by seanpianka** [palfrey/actix_lambda#5]
I see ALB support, but nothing noted for API Gateway.

**Comment by palfrey** [comment]
Yup. IIRC I couldn't figure out how to get API gateway working without also using ALB to serve the full [Dawn](https://github.com/palfrey/dawn) site which was my original target for this, so I didn't go down that route. I'm not really doing any further development here, but if you want to submit PRs to add API gateway support I'd be interested.

**Comment by seanpianka** [comment]
I was planning to deploy an AWS Lambda function which would serve an actix_web application, but purely for a REST API which returns JSON. I do not need to serve static assets, so an integration with API Gateway should be doable. Due to time constraints, I will be unable to provide a PR for this in the near-term.
